### PR TITLE
Add skipped tests to summary report

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers

@flopezag @jframos 
#### Description

Solves bug 4933
#### Testing

Verified against attached data files.

### CODE REVIEW

**Comment by jframos** [comment]
LGTM

File: fiware-region-sanity-tests/commons/results_analyzer.py [hunk 1]
```diff
@@ -47,24 +56,22 @@ def get_results(self):
...
-        print "{} [Tests: {}, Errors: {}, Failures: {}, Skipped: {}]".format(
-            testsuite.getAttribute("name"), testsuite.getAttribute("tests"), testsuite.getAttribute("errors"),
-            testsuite.getAttribute("failures"), testsuite.getAttribute("skip"))
-
-        # Total Tests
-        self.test += int(testsuite.getAttribute("tests"))
-
-        # Total Fail and Error Tests
-        self.failures += int(testsuite.getAttribute("failures")) + int(testsuite.getAttribute("errors"))
+        # Print a summary of the test results
```
Comment (flopezag) [inline fiware-region-sanity-tests/commons/results_analyzer.py:59]: And the testsuite.getAttribute("name")?

Comment (pratid) [inline fiware-region-sanity-tests/commons/results_analyzer.py:59]: Currently, get("name") gives "nosetests", so it is completely useless in this summary.

Comment (flopezag) [inline fiware-region-sanity-tests/commons/results_analyzer.py:59]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**